test/generic_operator_validation.jl

```diff
@@ -0,0 +1,45 @@
+using DiffEqOperators
+
+n = 100
+x=0.0:0.01:2π
+xprime = x[2:(end-1)]
+dx=diff(x)
+y = exp.(π*x)
+y_im = exp.(π*im*x)
+yim_ = y_im[2:(end-1)]
+y_ = y[2:(end-1)]
+
+
+
+for dor in 1:6, aor in 1:8
+
+    D1 = DerivativeOperator{Float64}(dor,aor,dx[1],length(x))
+    D2 = DiffEqOperators.FiniteDifference{Float64}(dor,aor,dx,length(x))
+    D = (D1,D2)
+    @test_broken convert(Array, D1) ≈ convert(Array, D2)
```

Comment:
while testing operators against operators is good, at some point we need to hard code some solutions to check against known analytical cases.